website/blog/2024-04-22-0.74-yoga-improvements-preparing-for-the-new-architecture.md

```diff
@@ -0,0 +1,437 @@
+4. Note that `soundName` will be null when it is accessed on a `Notification` returned from `getScheduledLocalNotifications()` and `getDeliveredNotifications()`.
+
+</details>
+
+### Other Breaking Changes
+
+These are some of the key breaking changes in 0.74. Please consult the [full changelog](https://github.com/facebook/react-native/blob/main/CHANGELOG.md) for the complete list of breaking changes.
+
+- Remove web performance logging from `GlobalPerformanceLogger` ([#41299](https://github.com/facebook/react-native/pull/41299)).
+- Make `start/end` in styles always refer to writing direction ([#42251](https://github.com/facebook/react-native/pull/42251)).
+- **Android**: Removal of `JSIModule*` from `FabricUIManagerProvider` ([#42059](https://github.com/facebook/react-native/pull/42059)).
+  - This API was unused in open source — use [TurboModules](https://github.com/reactwg/react-native-new-architecture/blob/main/docs/turbo-modules.md) instead.
+- **Android**: Deprecation of `UIManagerModule.showPopupMenu` and `UIManagerModule.dismissPopupMenu` ([#42441](https://github.com/facebook/react-native/pull/42441))
+  - This API has been moved to the `@react-native/popup-menu-android` NPM package and will be removed in 0.75
+- **iOS**: Delete `configFilename` and `configKey` arguments from iOS codegen CLI ([#41533](https://github.com/facebook/react-native/pull/41533)).
+- **iOS**: Changed how the [`bundleURL`](https://github.com/facebook/react-native/pull/43994) is handled.
+  - Before, this was set when React Native was started in an instance variable and it was not possible to update it. Now, it takes a function (usually [`bundleURL`](https://github.com/facebook/react-native/blob/0.74-stable/packages/react-native/template/ios/HelloWorld/AppDelegate.mm#L22-L29) in AppDelegate) so the URL is re-evaluated when needed. This unblocks the possibility to update `bundleURL` across refreshes. This should affect your app only if you were changing the `bundleURL` variable after the app is started. In this case, move the logic that updates the variable to the `bundleURL` function in the `AppDelegate`.
+
+## Acknowledgements
+
+React Native 0.74 contains over [1673 commits](https://github.com/facebook/react-native/compare/v0.73.6...v0.74.0) from 57 contributors. Thanks for all your hard work!
+
+## Upgrade to 0.74
+
+Please use the [React Native Upgrade Helper](https://react-native-community.github.io/upgrade-helper/) to view code changes between React Native versions for existing projects, in addition to the [Upgrading docs](/docs/upgrading).
+
+To create a new project:
+
+```sh
+npx react-native@latest init MyProject
+```
+
+If you use Expo, React Native 0.74 will be supported in Expo SDK 51.
+
+:::info
+0.74 is now the latest stable version of React Native and **0.71.x moves to unsupported**. For more information see [React Native's support policy](https://github.com/reactwg/react-native-releases#releases-support-policy).
```

Comment:
```suggestion
0.74 is now the latest stable version of React Native and **0.71.x moves to unsupported**. For more information see [React Native's support policy](https://github.com/reactwg/react-native-releases#releases-support-policy).
We will do a last end-of-life update of 0.71 at the beginning of May.
```

Given the privacy manifest issue from Apple, it makes sense to call out that we will make a last update to 0.71 with the last picks in May. Thoughts?

Reply:
Yup that's fair 👍